### PR TITLE
Include Codiva.io in the list of compilers

### PR DESCRIPTION
**Pull request by jayaprabhakar**
Codiva online compiler and IDE supports C, C++ and Java. 

It has syntax highlighting, continuous compilation, and works on mobile.

### CODE REVIEW

**Review by arnemertz — CHANGES_REQUESTED** [review]
Thanks for the PR! The Clang version is 3.5, and you changed the alternate text for the badge, but not the badge itself, it is still Clang-4.0-008080.svg
Could you change it to Clang-3.5?

You also note the compilation every few keystrokes. I could not verify it, i.e. it does compile only if I hit "run" (not logged in). Can you describe how to activate the feature?

**Comment by jayaprabhakar** [comment]
How do I find the badge image for the clang-3.5 version?

You may have to click 'Edit' to start editing. For other's project it always starts in readonly mode.

1. Try this project [https://www.codiva.io/p/0a5bebbd-210b-412e-aa5f-e4b029fa08c8](https://www.codiva.io/p/0a5bebbd-210b-412e-aa5f-e4b029fa08c8)
1. Click **Edit**
1. You will see 2 compilation errors.
1. Try fixing the errors.
    1. Insert the ';'
    1. correctly define the variable 'c'

File: index.md [hunk 1]
```diff
@@ -32,6 +33,18 @@ Besides mere compilation, most online compilers also execute the compiled progra
 
 ## The Compilers 
 
+### [Codiva.io](https://www.codiva.io)
+![C++14](https://img.shields.io/badge/C++-14-blue.svg)
+![Clang 3.5](https://img.shields.io/badge/Clang-4.0-008080.svg)
```
Comment (arnemertz) [inline index.md]: use `Clang 3.5` instead

**Comment by arnemertz** [comment]
The badge images are created from their URL, i.e. use another name and the image is different. You copied the use of `https://img.shields.io/badge/Clang-4.0-008080.svg` which is the Clang 4.0 badge, use `https://img.shields.io/badge/Clang-3.5-008080.svg` instead to make it a Clang 3.5 badge. More details here under "Your Badge" almost at the end of the page: https://shields.io/

**Comment by arnemertz** [comment]
Thanks @jayaprabhakar!

**Comment by jayaprabhakar** [comment]
Thanks @arnemertz. 
I see that you maintain another site https://arne-mertz.de/2017/05/online-compilers/ 
Would you mind including Codiva in that list?

**Comment by arnemertz** [comment]
That other one is a post that isn't maintained and explicitly points to this site.

**Pull request merged**